content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -377,20 +377,13 @@ const getUsersByRelevance = await prisma.user.findMany({
 
 <Admonition type="info">
 
+This feature is generally available in version `4.16.0` and later. It was first added as a [Preview feature](/concepts/components/preview-features) in version [`4.1.0`](https://github.com/prisma/prisma/releases/tag/4.1.0).
+
 Note: Prisma Client does not support this feature for MongoDB.
```

Comment:
I think we usually split the version info and feature limitations in different admonitions.